src/App.tsx

```diff
@@ -145,7 +145,7 @@ const App = () => {
 
   return (
     <>
-      <Helmet>
+      <Helmet defaultTitle={t('common:page_title')} titleTemplate={`%s - ${t('common:page_title')}`}>
```

Comment:
defaultTitle funker egentlig dårlig med lazy loading av komponenter i AppRoutes, siden det vil bli et mellomsteg hvor tittelen får defaultTitle siden den faktiske tittelen ikke er lastet på forhånd
tittel blir feks:
`Startside - NVA` -> `NVA` -> `Mine registreringer - NVA`
i stedet for:
`Startside - NVA` -> `Mine registreringer - NVA`

Hvis vi fjerner defaultTitle ser det mye bedre ut - men da må vi selv passe på at alle sider alltid har en tittel. hva er best?

Reply:
jeg tenker dette er godt nok som MVP. så får vi heller fjerne defaultTitle hvis det blir tatt opp. Uenig i at `/logout` burde være en snackbar. Det er veldig nice å kunne initiere/force logout via URL

Reply:
hvis man går til /logout nå får man bare "Du er logget ut" og en link til startsiden, selv om man fortsatt er innlogget :p 
enig i at det hadde vært fint å kunne trigge utlogging via URL, på samme måte som vi har for login, men når man har logget ut syns jeg det er bedre å redirecte til startsiden enn en side som bare sier "du har logget ut"

Reply:
oh, sant! blandet sikkert med `/login`. Ja, da kan vi heller redirecte til forsiden ja. tror AWS trenger logout-ruten i configen der, men skal vel være enkelt å endre